includes/Traits/Assets.php

```diff
@@ -123,7 +169,14 @@ public function register_style( $style_handle, array $style_dependencies = [] )
 	 * @return void
 	 */
 	public function enqueue_style( $style_handle, array $style_dependencies = [] ) {
-		$this->register_style( $style_handle, $style_dependencies );
+		$asset = $this->register_style( $style_handle, $style_dependencies );
+
+		// Enqueue chunks of $style_handle.
+		$style_chunks = $asset['css'];
+		foreach ( $style_chunks as $chunk ) {
+			wp_enqueue_style( $chunk );
+		}
```

Comment:
This is redundant and not needed.

If you enqueue a stylesheet, all its dependencies will be enqueued as well.

And since you already added all chunks to `$style_dependencies` in `register_style()` above, they will be loaded as expected.